vision/unit_tests/test_image.py

```diff
@@ -36,26 +36,26 @@ def _make_one(self, *args, **kw):
     def test_image_source_type_content(self):
         image = self._make_one(CLIENT_MOCK, content=IMAGE_CONTENT)
 
-        _AS_DICT = {
+        _as_dict = {
```

Comment:
Since you're editing here, you could add a trailing comma (if it pleases you)?

```python
_as_dict = {
    'content': B64_IMAGE_CONTENT',
}
# or go one-line
_as_dict = {'content': B64_IMAGE_CONTENT'}
```